pkg/portfwd/client.go

```diff
@@ -64,9 +58,6 @@ func HandleUDPConnection(ctx context.Context, client *guestagentclient.GuestAgen
 		buf := make([]byte, 65507)
 		for {
 			n, addr, err := conn.ReadFrom(buf)
```

Comment:
> Callers should always process
> // the n > 0 bytes returned before considering the error err.

https://pkg.go.dev/net#PacketConn

Seems this code is wrong.

Reply:
Correct, this is a bug in the code, but it is not related this change.

Reply:
Opened https://github.com/lima-vm/lima/issues/2970

Reply:
Should be fixd by #2971, thanks for pointing out this isue!